.ci.yaml

```diff
@@ -4620,6 +4620,58 @@ targets:
         ["devicelab", "hostonly", "mac"]
       task_name: flutter_tool_startup__macos
 
+  - name: Linux flutter_packaging_beta
+    recipe: packaging_v2/packaging_v2
+    timeout: 60
+    scheduler: release
+    bringup: true
+    enabled_branches:
+      - beta
+    properties:
+      task_name: flutter_packaging_beta
```

Comment:
Should these names be postfixed with OS?

Reply:
Do we do something with postfixes?

Reply:
I don't think so, but the other names in the file are setup in that convention, e.g. line 4612

Reply:
aaah, no, many of the values we are providing here are not needed but not sure if removing them will make TESTONWERS unhappy.